apps/auth/mix.exs

```diff
@@ -4,8 +4,7 @@ defmodule Auth.MixProject do
   def project do
     [
       app: :auth,
-      version: "0.1.0",
-      build_path: "../../_build",
+      version: "0.1.1/../_build",
       config_path: "../../config/config.exs",
```

Comment:
typo here if you look at the diff.

should be 
```elixir
version: "0.1.1",
build_path: "../../_build",
config_path: "../../config/config.exs",
...
```

The build path line was merged into the version line in a typo